docs/cellxgene_census_schema.md

```diff
@@ -872,8 +881,185 @@ Cell metadata MUST be encoded as a `SOMADataFrame` with the following columns:
 </tbody>
 </table>
 
+### Census Spatial Sequencing Data – `census_obj["census_spatial_sequencing"][organism]` – `SOMAExperiment`
+
+Only Visium and Slide-seq are supported for spatial data. [See the "assays included" section above](assays).
+
+Spatial data for *Homo sapiens* MUST be stored as a `SOMAExperiment` in `census_obj["census_spatial_sequencing"]["homo_sapiens"]`.
+
+Spatial data for *Mus musculus* MUST be stored as a `SOMAExperiment` in `census_obj["census_spatial_sequencing"]["mus_musculus"]`.
+
+For each organism the `SOMAExperiment` MUST contain the following:
+
+* Cell metadata – `census_obj["census_spatial_sequencing"][organism].obs` – `SOMADataFrame`
+* Non-spatial data  –  `census_obj["census_spatial_sequencing"][organism].ms` – `SOMACollection`. This `SOMACollection` MUST only contain one `SOMAMeasurement` in `census_obj["census_spatial_sequencing"][organism].ms["RNA"]` with the following:
+  * Matrix  data –  `census_obj["census_spatial_sequencing"][organism].ms["RNA"].X` – `SOMACollection`. It MUST contain exactly one layers:
+    * Count matrix – `census_obj["census_spatial_sequencing"][organism].ms["RNA"].X["raw"]` – `SOMASparseNDArray`
+  * Feature metadata – `census_obj["census_spatial_sequencing"][organism].ms["RNA"].var` – `SOMAIndexedDataFrame`
+  * Feature dataset presence matrix – `census_obj["census_spatial_sequencing"][organism].ms["RNA"]["feature_dataset_presence_matrix"]` – `SOMASparseNDArray`
+* Obs to spatial data mapping:
+  * Obs to spatial data – `census_obj["census_spatial_sequencing"][organism].obs_spatial_presence` – `SOMADataFrame`
+* Spatial data  –  `census_obj["census_spatial_sequencing"][organism].spatial` – `SOMACollection`.
+  * Spatial Scenes with spatial data  –  `census_obj["census_spatial_sequencing"][organism].spatial[scene_id]`  – `SOMAScene`.  There will be as many as Spatial Scenes as  spatial datasets. Each`SOMAScene` MUST contain the following:
```

Comment:
I am not familiar with the term "scene". Is this commonly used? Or else would be nice if it was defined in "Definitions" section of this document.

Reply:
I am having trouble finding a concise definition, but I believe it's a common term in computer graphics. There a scene would be the entire thing being rendered. E.g. a scene graph is a tree representation of all the objects to render.

In the SOMA specification, this is defined as:

> SOMAScene: a specialization and extension of SOMACollection that stores spatially resolved data that can be registered to a single coordinate space.

So I could do:

> scene: a collection of spatially resolved data that can be registered to a single coordinate space

?

Reply:
I see, yes, that definition looks good to me. Thanks!